web/src/routes/(user)/map/+page.svelte

```diff
@@ -43,6 +40,39 @@
 
   $: $featureFlags.map || goto(AppRoute.PHOTOS);
 
+  async function handleModeSwitch(mode: 'dark' | 'light') {
+    switch (mode) {
+      case 'dark':
+        style = await getDarkStyle();
+        break;
+      case 'light':
+        style = await getLightStyle();
+        break;
+    }
+  }
+
+  async function getLightStyle() {
+    const url = $serverConfig.mapStyles.find((style) => style.default === 'light')?.url;
+    if (url) {
+      const { data } = await axios.get<StyleSpecification>(url);
```

Comment:
Do we need to fetch the data or can we just pass maplibre the style URL?

Reply:
Oh I think you're right, totally missed that. Thanks!